src/style.css

```diff
@@ -0,0 +1,87 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+form {
```

Comment:
it is better to use class selectors instead of tag ones 
Apple ALL OVER the page, please